src/game.js

```diff
@@ -55,18 +57,18 @@ export class Game {
    * @returns {CSSStyleRule[]}
    */
   getStyleBySelectorAll(target, exact = true) {
+    // TODO This should NOT be in here!
```

Comment:
`getStyleBySelector` should not be in the `Game` class! This should be in its own `util` file or special DOM manipulation class.